recipes-sota/aktualizr/aktualizr-device-prov-creds.bb

```diff
@@ -6,13 +6,20 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7
 
 inherit allarch
 
-# WARNING: it is NOT a production solution. The secure way to provision devices
+# WARNING: This is NOT an ideal solution. The secure way to provision devices
 # is to create certificate request directly on the device (either with HSM/TPM
 # or with software) and then sign it with a CA stored on a disconnected machine.
 
-DEPENDS = "aktualizr aktualizr-native"
+# We need to get the config files from the aktualizr-host-tools package built by
+# the aktualizr (target) recipe.
+DEPENDS = "aktualizr aktualizr-native openssl-native"
 ALLOW_EMPTY_${PN} = "1"
 
+# If the config file from aktualizr used here is changed, you will need to bump
```

Comment:
That sounds dangerous. Would we lose a lot if we remove allarch from these recipes?

Reply:
That is the other option. But then we have to rebuild the recipes every time aktualizr changes, regardless of whether it actually affects these recipes. I opted for the case that is a bit painful in rare situations over the annoyance of slightly longer builds in the general case.

Reply:
Ok I see. I'm not too worried about the "painful" part but that it's hard to notice that the files have changed on bitbake's side and that it would cause wrong builds.

Reply:
> it's hard to notice that the files have changed on bitbake's side and that it would cause wrong builds.

Yes, agreed. I hoped the comment would be a hint, but it is a concern. Originally I was thinking about just removing `allarch` from those recipes, but it seemed like an unnecessary performance hit, so I went this way instead. Do you think the risk of unexpected build problems is too great?